pallets/spaces/src/benchmarking.rs

```diff
@@ -0,0 +1,87 @@
+//! Spaces pallet benchmarking.
+
+use super::*;
+
+use frame_benchmarking::benchmarks;
+use frame_benchmarking::whitelisted_caller;
+use frame_support::ensure;
+use frame_support::pallet_prelude::Get;
+use frame_system::RawOrigin;
+use sp_std::vec;
+
+use crate::Config;
+use crate::types::*;
+
+use pallet_permissions::{
+    default_permissions::DefaultSpacePermissions,
+    SpacePermissionSet,
+};
+
+
+fn dummy_space_content() -> Content {
+    Content::IPFS(b"QmRAQB6YaCaidP37UdDnjFY5aQuiBrbqdyoW1CaDgwxkD4".to_vec())
+}
```

Comment:
Replace with `subsocial_support::mock_functions::valid_content_ipfs`.

Also, you can refactor `mock_functions` mod in subsocial-support as you need, because basically it was added there to be then used in tests and benchmarks.